redaxo/src/core/pages/system.log.php

```diff
@@ -77,6 +77,12 @@
 $n['field'] = '<button class="btn btn-delete" type="submit" name="del_btn" data-confirm="' . rex_i18n::msg('delete') . '?">' . rex_i18n::msg('syslog_delete') . '</button>';
 $formElements[] = $n;
 
+if ($url = rex_editor::factory()->getUrl($logFile, 0)) {
+    $n = [];
+    $n['field'] = '<a class="btn btn-save" href="'. $url .'">' . rex_i18n::msg('system_editor_open_file', basename($logFile)) . '</a>';
```

Comment:
@tbaddade können/sollten wir hier eine klasse `btn-open` o.ä. neu definieren? oder gibt es bereits eine die ich hier stattdessn nehmen sollte?

`btn-save` wird aktuell nur verwendet damit die optik ungefähr passt.

Reply:
da wir vermutlich diese "im editor öffnen" button noch an weiteren stellen bekommen, macht es ggf. sinn einen `btn-editor-open` (name natürlich noch frei wählbar) verwendet werden, der sich durch ein entsprechendes icon ein wenig abhebt bzw. wiedererkennbar wird.

Reply:
Was ist mit denen, die nicht lokal entwickeln? Könnte man da noch einen Download-Button als Shortcut einfügen?

Reply:
Dafür ist geplant, wenn der editor an sich gereift ist, ein remote-mapping einzubauen.


D.h. Man stellt irgenwo einen lokalen pfad im redaxo/system ein und das feature funktioniert genauso wie bei lokaler Entwicklung

Reply:
Da garantiere ich, dass eine Vielzahl an REDAXO-Entwicklern das erstmal nicht nutzen werden/können, weil sie z.T. ausschließlich online entwickeln - ohne, dass das jetzt ein gutes / zeitgemäßes Pattern ist. REDAXO tut dem oft genüge, indem es Eingabemasken bereitstellt (bspw. bei Templates und Modulen) - hier nicht.

Und in diesem Fall kann ich mir gar nicht vorstellen, wie lokal das Log geöffnet werden soll, wenn es sich naturgemäß remote unterscheidet.

Reply:
Siehe https://github.com/redaxo/redaxo/issues/1888

Reply:
Noch kurz als nachtrag: durch den download von dateien wird man dann nur readonly inhalte sehen können. Das hauptfeature der api, dass man direkt die files im editor öffnet und bearbeiten kann - wird mit dem remote only workflow nicht möglich sein